[2024-06-02T16:59:27.053+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: concurrent_dag.wait2 manual__2024-06-02T16:57:15.855862+00:00 [queued]>
[2024-06-02T16:59:27.081+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: concurrent_dag.wait2 manual__2024-06-02T16:57:15.855862+00:00 [queued]>
[2024-06-02T16:59:27.082+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-06-02T16:59:27.146+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): wait2> on 2024-06-02 16:57:15.855862+00:00
[2024-06-02T16:59:27.167+0000] {standard_task_runner.py:60} INFO - Started process 3621 to run task
[2024-06-02T16:59:27.183+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'concurrent_dag', 'wait2', 'manual__2024-06-02T16:57:15.855862+00:00', '--job-id', '305', '--raw', '--subdir', 'DAGS_FOLDER/pool_dag.py', '--cfg-path', '/tmp/tmpfnle_n_r']
[2024-06-02T16:59:27.211+0000] {standard_task_runner.py:88} INFO - Job 305: Subtask wait2
[2024-06-02T16:59:27.248+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-02T16:59:27.448+0000] {task_command.py:423} INFO - Running <TaskInstance: concurrent_dag.wait2 manual__2024-06-02T16:57:15.855862+00:00 [running]> on host ac6c883b25f7
[2024-06-02T16:59:27.726+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='concurrent_dag' AIRFLOW_CTX_TASK_ID='wait2' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T16:57:15.855862+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-02T16:57:15.855862+00:00'
[2024-06-02T17:00:27.789+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-06-02T17:00:27.809+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=concurrent_dag, task_id=wait2, execution_date=20240602T165715, start_date=20240602T165927, end_date=20240602T170027
[2024-06-02T17:00:27.879+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T17:00:27.904+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
